content/messaging/contents.lr

```diff
@@ -151,21 +157,23 @@ There is much safer alternatives to Skype for video calling - please use them!
 
 ##### Konversation
 **Warning:**
-See [KDE warnings](#KDEWarnings).
+See [KDE warnings](#KDE).
 
 Konversation is an open-source IRC client built for the KDE environment.
 It is currently maintained in the KDE Extragear Network module, which means that it has its own release cycle which is independent from the main KDE applications.
 Konversation is released under the terms of the GNU General Public License.
 
 ##### Kopete
 **Warning:**
-See [KDE warnings](#KDEWarnings).
+See [KDE warnings](#KDE).
 
 Kopete is an open-source, multi-protocol instant messaging client released in 2001 in the wake of ICQ blocking Licq from their network.
 It is part of the KDE software compilation, however it is capable of running in numerous environments including Gnome and XFCE.
 
+----
+
 #### Additional warnings
-##### KDE warnings
+##### KDE Warnings<a name="KDE"></a>
```

Comment:
I was just thinking about anchors! I think it would be nice if lektor did like trac, and automatically generated an anchor with a heading. This become super clean and practical, for example, to refer to glossary terms in the current glossary: https://trac.torproject.org/projects/tor/wiki/doc/community/glossary#WebBrowser

Reply:
OK, I have opened https://trac.torproject.org/projects/tor/ticket/28501 to get the ​https://www.getlektor.com/plugins/lektor-markdown-header-anchors/ plugin that gives anchors...